app/interactors/create_activity.rb

```diff
@@ -0,0 +1,9 @@
+class CreateActivity
+  include Interactor
+
+  delegate :entity, :action, :activity, :owner, :params, to: :context
```

Comment:
@dmitrytrager create_activity is a built in method supplied by the gem public_activity, gets added to any models that you include PublicActivity::Common on.